plugins/content_distribution/providers/tvinci/lib/api/KalturaTvinciDistributionJobProviderData.php

```diff
@@ -83,26 +93,28 @@ public function __construct(KalturaDistributionJobData $distributionJobData = nu
 		{
 			$this->xml = $feedHelper->buildDeleteFeed();
 		}
+		KalturaLog::debug("XML Constructed by the Tvinci feed helper :{$this->xml}");
+
 	}
 
 	private function initPlayManifestUrls($entry, $feedHelper)
 	{
 		$videoAssetDataMap = array();
 
-		// If the following field is defined, it will override the below hardcoded defaults
-		$videoAssetConfigLines = $feedHelper->getSafeFieldValue(TvinciDistributionField::VIDEO_ASSETS_CONFIGURATION, null);
-		if ( $videoAssetConfigLines )
-		{
-			// The format is a comma separated array of these compounds: "name:protocol:tag:ext"
-			// E.g.: "name:protocol:tag:ext,name:protocol:tag:ext,name:protocol:tag:ext"
-			$configLines = explode(',', $videoAssetConfigLines);
-			foreach ( $configLines as $configLine )
-			{
-				$vad = explode(':', $configLine);
-				$videoAssetDataMap[] = array($vad[0], $vad[1], $vad[2], $vad[3]);
-			}
-		}
-		elseif ( $feedHelper->schemaId() == 1 )
+//		// If the following field is defined, it will override the below hardcoded defaults
```

Comment:
Why is it commented out? If you don't need it, delete it, and delete any reference to $videoAssetDataMap later in this method.